recipes/monotonicnetworks/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "monotonicnetworks" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/monotonicnetworks-{{ version }}.tar.gz
+  sha256: 5bf2a188f2272f0fd8113ddc91da457d140ddfd1a145917136deb536113017af
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - pytorch >=1.8
```

Comment:
pytorch is only needed at run time. It's not required to build the package. You can remove it.

Reply:
Removed.